lib/Common/Common/Jobs.cpp

```diff
@@ -464,6 +465,9 @@ namespace JsUtil
         // Do nothing
     }
 
+// Xplat-todo: revive BackgroundJobProcessor- we need this for the JIT
+#if ENABLE_BACKGROUND_JOB_PROCESSOR
```

Comment:
nit: Hard to find out where this #if ends. Add to #endif // ENABLE_BACKGROUND_JOB_PROCESSOR